GG/src/DropDownList.cpp

```diff
@@ -118,6 +121,9 @@ class ModalListPicker : public std::enable_shared_from_this<ModalListPicker>, pu
     const size_t        m_num_shown_rows;
     const DropDownList* m_relative_to_wnd;
     bool                m_dropped; ///< Is the drop down list open.
+
+    /** Should the list wnd scroll when dropped? */
```

Comment:
to be more clear (and consistent with the method comments), "when" --> "only when"

But also, I'll let myself mention a slight generalization here, which would not in any sense be needed but which I think would be kind of nice to do now, or at least I'll mention it now so it's kept in mind -- namely, if this variable and the related methods were renamed somewhat more generically along the lines of "m_restrict_mousewheel_scroll".

That would just be a preparatory change, looking forward to when we could add back the enablement condition you had before, ctrl bring pressed, so that mousewheel scrolling would be enabled if any of the three conditions were met-- the restriction bool is not set, or the dropdown is dropped, or ctrl is being pressed.  I would not propose actually adding back the ctrl enablement at this time, because I believe the ProductionWnd dropdown handling is right now dependent on actual clicks being made in order for changes to the dropdown to get processed properly, so that would need a bit of work done at the same time.